packages/fast-development-site-react/src/utilities/form-child-from-examples-factory.ts

```diff
@@ -0,0 +1,22 @@
+import { IFormChildOption } from "../components/";
+
+export interface IExample {
+    name: string;
+    component: any;
+    schema: JSON;
+    data: any[];
+}
+
+export default function componentFactory(examples: IExample): IFormChildOption[] {
+    const formChildOptions: IFormChildOption[] = [];
```

Comment:
you can just use `.map` to avoid creating this variable:

```
return Object.keys(examples).map((key: string) => {
    return {
        // ....
    };
});
```

Reply:
fixed